api/resource_releases.go

```diff
@@ -40,7 +40,45 @@ func (c *Client) GetAppReleases(ctx context.Context, appName string, limit int)
 	return data.App.Releases.Nodes, nil
 }
 
-func (c *Client) GetAppRelease(ctx context.Context, appName string, id string) (*Release, error) {
+func (c *Client) GetAppReleasesMachines(ctx context.Context, appName string, limit int) ([]Release, error) {
+	query := `
+		query ($appName: String!, $limit: Int!) {
+			app(name: $appName) {
+				releases: releasesUnprocessed(first: $limit) {
```

Comment:
nice! I didn't know graphql supported renaming the field. TIL

Reply:
me neither! just wanted to find a way to _not_ cram new stuff into the huge query data struct haha